File: common/GlobalConstant.py
__author__ = 'xiaodan'
import pyodbc
from datetime import datetime
import QDate
from local_env import DBCONN_WIND
from local_env import BASE_DIR, CODE_DIR, DATA_DIR
# %% header
# % store all constant variables that can be accessed globally
# % date: 9/06/2015

#locations



#test start and end date
TestStartDate = datetime(2008, 12, 31)
TestEndDate = datetime(2014, 12, 31)
#exchange trading dates
TradingDates = QDate.GetAllTradingDays()

#holding structure
Holding = ['StockID', 'Weight', 'Shares', 'MarketValue']

File: matlab/common/GlobalConstant.m
%% header
% store all constant variables that can be accessed globally
% date: 7/17/2015

%%

classdef GlobalConstant       
    properties (Constant)
        % locations
        BASE_DIR = 'd:\ChinaA\';
        CODE_DIR = 'd:\ChinaA\source\';
        DATA_DIR = 'd:\ChinaA\data\';
        
        % test start and end date
        TestStartDate = '20081231';
        TestEndDate = '20141231';
        
        % database connection
        DBCONN_WIND = database('WindDB','','','Vendor','Microsoft SQL Server','Server','localhost','AuthType','Windows','PortNumber',1433);
        
        % exchange trading dates
        TradingDates = QDate.GetAllTradingDays();
        
        % holding structure
        Holding = {'StockID', 'Weight', 'Shares', 'MarketValue'};
    end
end
